### PR TITLE
Veraison perf evaluation

### PR DESCRIPTION
**Pull request by setrofim**

### CODE REVIEW

File: perf/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Requirements
+
+This requires `locust` to generate loads and `jupyter-lab` to analyse the
+results. Both are Python applications, and so may be installed via `pip`, or
+through your system's package manager.
+
+## Basic flow
+
+1. Run loads an collect data:
```
Comment (thomas-fossati) [inline perf/README.md:9]:
```suggestion
1. Run loads and collect data:
```

File: scripts/extract-corim [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# This script is used to extract the claims structure (serialized CBOR) from COSE tokens.
+# On Arch, you need in stall yq and ruby-cbor-diag (AUR) packages.
+# On Ubuntu, install cbor-diag gem; yq seems to only be available via *shudder* snap...
```
Comment (thomas-fossati) [inline scripts/extract-corim:4]: 🤣

**Review by thomas-fossati — APPROVED** [review]
awesome, thanks

**Comment by yogeshbdeshpande** [comment]
@setrofim : Thanks for the PR: in general LGTM! One question: Are the files under notebook autogenerated, if so, are they strictly required to be submitted inside github, or we can fix a configuration for the required packages and they can be generated on each machine for that config?

**Comment by setrofim** [comment]
> @setrofim : Thanks for the PR: in general LGTM! One question: Are the files under notebook autogenerated, if so, are they strictly required to be submitted inside github, or we can fix a configuration for the required packages and they can be generated on each machine for that config?

No, they're not auto generated.  These are the Jupter notebooks (specifically-wrapped Python (in this case) code that is written by hand).

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Comment by yogeshbdeshpande** [comment]
> > @setrofim : Thanks for the PR: in general LGTM! One question: Are the files under notebook autogenerated, if so, are they strictly required to be submitted inside github, or we can fix a configuration for the required packages and they can be generated on each machine for that config?
> 
> No, they're not auto generated. These are the Jupter notebooks (specifically-wrapped Python (in this case) code that is written by hand).

Thanks for the clarification 👍

**Pull request merged**